wfe2/verify.go

```diff
@@ -548,6 +550,20 @@ func (wfe *WebFrontEndImpl) validSelfAuthenticatedPOST(
 	return payload, pubKey, nil
 }
 
+// validSelfAuthenticatedPOST checks that a given POST request has a valid JWS
+// using `validSelfAuthenticatedJWS`.
+func (wfe *WebFrontEndImpl) validSelfAuthenticatedPOST(
+	request *http.Request,
+	logEvent *requestEvent) ([]byte, *jose.JSONWebKey, *probs.ProblemDetails) {
+	// Extract the embedded JWK from the parsed JWS
```

Comment:
This comment appears to be in the wrong place, I feel like it should be above L564?